source/io/JavaScriptInvoke.cpp

```diff
@@ -150,7 +150,13 @@ VIREO_FUNCTION_SIGNATUREV(JavaScriptInvoke, JavaScriptInvokeParamBlock)
                 &errorClusterPtr->status,
                 &errorClusterPtr->code,
                 errorClusterPtr->source);
-            AddCallChainToSourceIfErrorPresent(errorClusterPtr, "JavaScriptInvoke");
+
+            if (isInternalFunction) {
+                AddCallChainToSourceIfErrorPresent(errorClusterPtr, (const char*)functionName->Begin());
```

Comment:
It would be nice to get the name of the property in the message.

Reply:
Good point. But if I'm not mistaken that will be part of the error message that is generated in the actual code dealing with the property node call.

Reply:
That would be the responsibility of the ASW JS implementing the property node behavior, not the JavaScriptInvoke node itself.